src/containers/VaccineInfo/VaccineInfo.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  DisplayText,
+  Layout,
+  MediaCard,
+  VideoThumbnail,
+} from "@shopify/polaris";
+import { useTranslation } from "react-i18next";
+import React from "react";
+
+export function VaccineInfo() {
+  const { t } = useTranslation();
+  return (
+    <Layout>
+      <Layout.Section>
+        <DisplayText size="extraLarge">Vaccine Information</DisplayText>
+        <DisplayText size="small">
+          We have collected these resources for you to get more information on
+          vaccines.
+        </DisplayText>
+      </Layout.Section>
+      <Layout.Section>
+        <MediaCard
+          portrait
+          title="COVID-19: How vaccines are developed"
+          primaryAction={{
+            content: "Learn more",
+            url: "https://www.youtube.com/watch?v=Y51ZgZCS8J0",
+          }}
+          description="This video explains the vaccine development process and how it is possible to achieve a COVID-19 vaccine, with significant domestic and international collaboration, in a much shorter period of time."
+        >
+          <VideoThumbnail
+            videoLength={204}
+            thumbnailUrl="https://img.youtube.com/vi/Y51ZgZCS8J0/0.jpg"
+            onClick={() => {
+              window.open("https://www.youtube.com/watch?v=Y51ZgZCS8J0");
+            }}
+          />
+        </MediaCard>
+        <MediaCard
+          portrait
+          title="How mRNA Vaccines Work - Simply Explained"
+          primaryAction={{
+            content: "Learn more",
+            url: "https://www.youtube.com/watch?v=WOvvyqJ-vwo",
+          }}
+          description="mRNA vaccines have to potential to end the COVID19 pandemic. How do they work? Are they safe? And how could they've been developed so quickly?"
+        >
+          <VideoThumbnail
+            videoLength={265}
+            thumbnailUrl="https://img.youtube.com/vi/WOvvyqJ-vwo/0.jpg"
+            onClick={() => {
+              window.open("https://www.youtube.com/watch?v=WOvvyqJ-vwo");
+            }}
+          />
+        </MediaCard>
+        <MediaCard
+          portrait
+          title="This Is Our Shot"
+          primaryAction={{
+            content: t("visitwebsite"),
+            url: "https://thisisourshot.ca/",
+          }}
+          description="Provides a broad array of vaccine information like information on
+            health and saftey, information for before, during and after your
+            vaccination, and live town halls by experts."
+        >
+          <img
+            alt="This Is Our Shot Logo"
+            width="100%"
+            height="100%"
+            style={{
```

Comment:
I haven't pulled this down, but just be aware these styles can blow an image up, and could make it fuzzy especially on high-res displays.